### PR TITLE
Fl13 update map

### PR DESCRIPTION
**Pull request by Danatlol**

### CODE REVIEW

**Comment by Danatlol** [comment]
[FL13_update_map](https://trello.com/c/O7jlTLrd/50-fl13updatemap)

File: src/app/search-places.service.ts [hunk 1]
```diff
@@ -13,23 +16,12 @@ export class SearchPlacesService {
...
-      type: type,
-      openNow: openNow
-    };
     const service = new google.maps.places.PlacesService(newmmap);
 
-    service.radarSearch(request, (results, status) => {
-      // console.log(results);
-      // console.log(status);
-
+    return <any>Observable.bindCallback(service.radarSearch.bind(service), (results, status) => {
```
Comment (Evittta) [inline src/app/search-places.service.ts]: try change any to another type

File: src/app/search-places.service.ts [hunk 2]
```diff
@@ -39,36 +31,32 @@ export class SearchPlacesService {
...
-        this.store.dispatch(new SearchPlaces.StopRadarSearch([]));
+        return [];
       }
     });
   }
 
-  clearPlaces() {
-    this.store.dispatch(new SearchPlaces.ClearPlacesList());
+  getCurrentLocationCoordinates() {
+    return <any>Observable.bindCallback(navigator.geolocation.getCurrentPosition.bind(navigator.geolocation), (position) => {
```
Comment (Evittta) [inline src/app/search-places.service.ts]: avoid long code lines

File: src/app/main/coord-picker/coord-picker.component.html [hunk 3]
```diff
@@ -43,27 +43,18 @@
...
-    [radius]="radius"
-    [fillColor]="'cornflowerblue'"
-    [circleDraggable]="false"
-    [editable]="false"
-    (circleClick)="onClick($event)"
-    ></agm-circle> -->
   </agm-map>
-  <mat-slider thumbLabel min="0" max="300" step="1" [(ngModel)]="radius" placeholder="qwe" color="primary"></mat-slider>
+
+  <mat-slider thumbLabel min="0" max="300" step="1" [value]="radius$ | async" placeholder="qwe" color="primary" (change)="onSearchRadiusChange($event)"></mat-slider>
```
Comment (Evittta) [inline src/app/main/coord-picker/coord-picker.component.html]: avoid long code lines

**Pull request merged**